File: RegistroTareasEntities/BDEntities/Tarea.cs

namespace RegistroTareasEntities.BDEntities
{
    public partial class Tarea
    {
        public int Id { get; set; }
        public int IdUsuario { get; set; }
        public string Nombre { get; set; } = null!;
        public string Descripcion { get; set; } = null!;
        public DateTime FechaInicio { get; set; }
        public DateTime FechaFin { get; set; }
        public int Horas { get; set; }
        public bool Completado { get; set; }
        public DateTime FechaCreacion { get; set; }
        public DateTime? FechaModificacion { get; set; }

        public virtual Usuario IdUsuarioNavigation { get; set; } = null!;
    }
}

File: TareasWebApp/src/app/classes/tarea.ts
export class Tarea {
    id!:number;
    idUsuario!:number;
    nombre!:string;
    descripcion!:string;
    fechaInicio!:Date;
    fechaFin!:Date;
    horas!:number;
    esCompletada:boolean = false;
    fechaCreacion!:Date;
    fechaModificacion!:Date;
}
